### PR TITLE
Adds error logging when exceptions are thrown.

### PR DESCRIPTION
**Pull request by randy-girard**

### CODE REVIEW

**Comment by randy-girard** [comment]
@knappe @ryanwi Replaced the other PR with this one, just for logging.

File: lib/emque/consuming/adapters/rabbit_mq/delayed_message_worker.rb [hunk 1]
```diff
@@ -68,9 +68,11 @@ def process_message(delivery_info, metadata, payload)
               )
               ::Emque::Consuming::Consumer.new.consume(:process, message)
               channel.ack(delivery_info.delivery_tag)
-            rescue StandardError => ex
-              if retryable_errors.any? { |error| ex.class.to_s =~ /#{error}/ }
-                retry_error(delivery_info, metadata, payload, ex)
+            rescue StandardError => exception
+              logger.error "#{log_prefix} #{exception.class}: #{exception.message}"
+              exception.backtrace.each {|bt| logger.error "#{log_prefix} #{bt}" }
```
Comment (knappe) [inline lib/emque/consuming/adapters/rabbit_mq/delayed_message_worker.rb]: Can you please pad this with spaces?  `{ |bt| ... }

File: lib/emque/consuming/adapters/rabbit_mq/error_worker.rb [hunk 2]
```diff
@@ -55,7 +55,9 @@ def retry_message(delivery_info, metadata, payload)
               )
               ::Emque::Consuming::Consumer.new.consume(:process, message)
               channel.ack(delivery_info.delivery_tag)
-            rescue StandardError
+            rescue StandardError => exception
+              logger.error "#{log_prefix} #{exception.class}: #{exception.message}"
+              exception.backtrace.each {|bt| logger.error "#{log_prefix} #{bt}" }
```
Comment (knappe) [inline lib/emque/consuming/adapters/rabbit_mq/error_worker.rb]: Here too.

File: lib/emque/consuming/adapters/rabbit_mq/worker.rb [hunk 3]
```diff
@@ -78,7 +78,9 @@ def process_message(delivery_info, metadata, payload)
               )
               ::Emque::Consuming::Consumer.new.consume(:process, message)
               channel.ack(delivery_info.delivery_tag)
-            rescue StandardError => ex
+            rescue StandardError => exception
+              logger.error "#{log_prefix} #{exception.class}: #{exception.message}"
+              exception.backtrace.each {|bt| logger.error "#{log_prefix} #{bt}" }
```
Comment (knappe) [inline lib/emque/consuming/adapters/rabbit_mq/worker.rb]: And here.

**Comment by randy-girard** [comment]
@knappe updated

**Pull request merged**